random/helpers.go

```diff
@@ -68,6 +68,30 @@ func RandomStrings(n int) []string {
 	return l
 }
 
+func RandomElementFromArray(arr []string) string {
+	return RandomElementsFromArray(arr, 1)[0]
+}
+
+func RandomElementsFromArray[T any](arr []T, count int) []T {
```

Comment:
This and the `shuffle` method below are kinda similar btw - wonder if we can remove `shuffle` then?

Reply:
Yes, makes sense. 
Have removed it.